flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/declarative/DeclarativeSchedulerNG.java

```diff
@@ -902,6 +912,39 @@ private void deploySafely(ExecutionVertex executionVertex) {
         private void handleDeploymentFailure(ExecutionVertex executionVertex, JobException e) {
             executionVertex.markFailed(e);
         }
+
+        @Override
+        public void newResourcesAvailable() {
+            int availableSlots = declarativeSlotPool.getFreeSlotsInformation().size();
+
+            if (availableSlots > 0) {
+                LOG.info(
```

Comment:
log as debug or maybe even trace; the period check shouldn't be too interesting.